snap/validate.go

```diff
@@ -399,6 +399,10 @@ func Validate(info *Info) error {
 	if err := ValidateLinks(info.OriginalLinks); err != nil {
 		return err
 	}
+	// TODO: Consider adding once cleaning of persisted invalid EditedLinks is available
+	//if err := ValidateLinks(info.EditedLinks); err != nil {
+	//        return err
+	//}
```

Comment:
adding this is a bit odd wrt the normalisation code we added

Reply:
Removed